stack-qa/recipes/role-redirector.rb

```diff
@@ -1,9 +1,3 @@
 include_recipe 'ies::role-generic'
 include_recipe 'nginx-app::server'
-
-node['deploy'].each do |application, deploy|
```

Comment:
Removed this as it will all happen in one recipe, without any OpsWorks deployment or application.